### ISSUE TITLE
reduced structure tests for fxl edupubs

### ISSUE THREAD

**Issue by mattgarrish** [IDPF/epubcheck#580]
Need to disable the section/heading tests in edu-structure.sch when an edupub document is fxl, either by the global rendition:layout setting or via a spine-level override.

**Comment by rdeltour** [comment]
OK. Can you confirm this applies to **all** the tests in `edu-structure.sch`?

**Comment by mattgarrish** [comment]
Yes, disable that schema in its entirety. All those tests either check for proper sectioning or use of headings/subtitles, and the paginated nature of FXL documents makes sectioning mostly impossible and heading enforcement nearly-so.

We have guidelines elsewhere for making FXL docs accessible, but checking those requirements isn't automatable.

**Comment by rdeltour** [comment]
OK good. I wasn't sure since the spec says:

> Due to the paginated nature of fixed layouts, Fixed-Layout Publications are exempt from the requirements in this section.

at the end of section 4.2, but it then seems it also applies to the following section 4.3.

**Comment by mattgarrish** [comment]
I'm going to open an issue to see if we should clarify heading rules for fxl, even if that just leads to a clearer exemption (maybe a subsection of 4.3 on fxl). 4.3 is really a continuation of 4.2, as 4.2 makes the case that every section has to have a heading and then 4.3 elaborates on how to apply headings to sections. That's the way we developed it, at least.

I can't really see right now how we'd add anything useful about fxl headings, though. Since we're not enforcing sectioning on fxl documents, it could be incredibly hard to provide reliable validation of headings, so I'm not sure what we coud enforce. An fxl page could have an h1 legitimately followed by an h2, for example, since sections aren't needed, but epubcheck could never know if the h2 was a mistaken subheading or not. You'd never really know if subsection, asides, etc. have out-of-sequence headings, either, as any given page is going to have limited context. But I'm not telling you anything you don't already know.

All I can think is that we could enforce that levels not be skipped if more than one ranked heading is present in a given document, but that's having to dig to justify rules. We might just want to encourage the practices we've outlined for reflowable without requiring them.
